todo_app/app.py

```diff
@@ -37,6 +43,12 @@ def writer_check(*args, **kwargs):
 def create_app():
     app = Flask(__name__)
     app.config.from_object(Config())
+    logger = logging.getLogger(__name__)
```

Comment:
I don't think you ever use this logger, so no harm but we might as well remove this line

Reply:
Use it in some of the cases? like `app.logger.info("Card (" + str(cards.find_one({"_id":ObjectId(_id)}).get("name")) + ") Completed")`